### DISCUSSION TITLE
Application for a Teachers benefits frozen

### DISCUSSION THREAD

**Discussion by buczbuska** [community/community#86731] [GitHub Education] [answered]
### Hi everyone,

Guys,
I have a problem with my application for the Teachers' Education application. I cannot finish my application. When I try to upload the photo of ID, nothing happens on the website, and everything is stuck in the Processing Application stage (the process is frozen on uploading the photo). Can you please help me with that?

Best
Bartek

**Comment by ilyaveer** [comment]
Hello!

I have a similar issue. I uploaded the necessary proof of employment, a jpeg of my contract (dimensions and file size within specified boundaries), but nothing happens when I press the "process my application" button. It seems to be hanging on "Processing Information" (see screenshot) and after hours of waiting I just cancelled it. This behavior happens in Safari, Chrome, and Firefox, so it doesn't seem to be browser dependent.

Any help would be greatly appreciated!

Cheers,
Ilya

<img width="977" alt="Screenshot 2024-01-17 at 15 14 37" src="https://github.com/community/community/assets/23030197/6ef05333-c940-4df7-b762-c8e8df89f6cc">

**Comment by buczbuska** [comment, marked as answer]
Hi Ilya,
For me, resizing the documents worked, and a lot of people have resolved
this issue in this way. Just resize the jpegs and make them really light
(like 200-400kb), and it should work!

Best
Bartek

śr., 17 sty 2024 o 15:23 Ilya Veer ***@***.***> napisał(a):

**Reply by ilyaveer** [reply]
Thanks, that worked! Although strange that a somewhat larger file that was still within boundaries (by far) was not accepted...

**Comment by buczbuska** [comment]
Yaaay! Yeah, it's weird XD

śr., 17 sty 2024 o 16:34 Ilya Veer ***@***.***> napisał(a):
